src/Buzzer.cpp

```diff
@@ -12,7 +12,6 @@ Buzzer::Buzzer(){
 }
 
 void Buzzer::soundTone(int frequency) {
-  if(Properties::DEBUGGING_ACTIVE) Serial.println("Buzzer has started sounding");
```

Comment:
Was it intentional that these two logging statements were removed instead of just changed into the new format?